### ISSUE TITLE
failed to init cuda platform

### ISSUE THREAD

**Issue by 2017monster2017** [FinMiner/FinMiner#16]
same issue before 2.2.4...… 
I don't even have the folder Nvidia/corporation/n...… etc....
 I cant use 2.2x.... any version

**Comment by FinMiner** [comment]
Did you try latest 2.3 version? What is the exact log? We have issues with "failed to init cuda platform" errors related to some bug in Nvidia SLI. Please see #7 and #8

**Comment by FinMiner** [comment]
Do you see "nvml.dll is probably missing" in log?
If so - please search for nvml.dll somewhere in Nvidia driver installation on your system and copy it inside FinMiner directory. If it works, please share the location of nvml.dll on your system. We try to get one from Program Files\NVIDIA Corporation\NVSMI\nvml.dll or current directory but both tries have failed as far as I understand.

**Comment by 2017monster2017** [comment]
Yes, did it (from computer folder to finminer folder), but is not working.
However the log that I have now it shows stoping after sentence :
CUDA driver version is 10.0, runtime version is 9.0

And nothing else.... and then close the program.

**Comment by FinMiner** [comment]
So at least something has changed. I've been thinking for some time what to do and have nothing better than checking single connected 1080ti, then two of them, etc. So the question is if finminer works in case of single GPU physically connected on you system? If not, what is in miner log? If yes, does it work with two GPUs?
